Decred Wallet/en.lproj/Localizable.strings

```diff
@@ -393,3 +393,29 @@
 "history" = "history";
 "settings" = "Settings";
 "help" = "help";
+
+/* Backup */
+
+"KeepInMind" = "Keep in mind";
+"ViewSeedPhrase" = "View seed phrase";
+"reminderLabel1" = "The <bold>33-word seed phrase</bold> is EXTREMELY IMPORTANT.";
+"reminderLabel2" = "Seed phrase is <bold>the only way</bold> to restore your wallet.";
+"reminderLabel3" = "It is recommended to store your seed phrase in a <green>physical format</green> (e.g. write down on a paper).";
+"reminderLabel4" = "It is highly discouraged to store your seed phrase in any <orange>digital format</orange> (e.g. screenshot).";
+"reminderLabel5" = "Anyone with your seed phrase <orange>can steal your funds</orange>. DO NOT show it to anyone.";
+
+"WriteDownSeedPhrase" = "Write down seed phrase";
+"WriteDownStep1_2" = "Step 1/2";
+"WriteDownStep2_2" = "Step 2/2";
+
+"WriteDownAll33Words" = "Write down all 33 words in the correct order.";
+"Your33word" = "Your 33-word seed phrase";
+"iHaveWroteDown" = "I have wrote down all 33 words";
+"Verify" = "Verify";
+"BackupSuccessTitle" = "Your seed phrase backup is verified";
+"BackupSuccessDesc" = "Be sure to store your seed phrase backup in a secure location.";
+"BackToOverview" = "Back to Overview";
+"FailedToVerify" = "Failed to verify. Please go through every words and try again.";
+"Verifying" = "Verifying...";
+"VerifySeedPhrase" = "Verify seed phrase";
+"SelectTheCorrectWordsToVerify" = "Select the correct words to verify.";
```

Comment:
There is a format(i.e "backupSuccessDesc" instead of "BackupSuccessDesc") we use here please follow them and copy and paste them in other language files